server/build.gradle

```diff
@@ -126,6 +129,11 @@ task copyWebApp(type: Copy) {
     doFirst {
         delete(dst)
     }
+
+    // gradle holds a lock even after the buildWebapp task is completed
```

Comment:
Question: which lock file is it?

Reply:
How about specifying the file?
reference: https://github.com/Kotlin/kotlinx-datetime/blob/master/core/build.gradle.kts#L295-L301